pages/generative-apis/how-to/use-structured-outputs.mdx

```diff
@@ -31,22 +31,19 @@ There are several ways to interact with language models:
 
 ## Types of structured outputs
 
-- **JSON mode** (schemaless):
+- **Structured outputs (schema mode)**:
+   - Type `{"type": "json_schema"}`
+   - This mode enforces a strict schema format, where the output adheres to the predefined structure.
+   - Supports complex types and validation mechanisms as per the [JSON schema specification](https://json-schema.org/specification/), including nested schemas composition (`anyOf`, `allOf`, `oneOf` etc), `$ref`, `all` types, and regular expressions.
+
+- **JSON mode** (Legacy method):
    - Type: `{"type": "json_object"}`
    - This mode is non-deterministic and allows the model to output a JSON object without strict validation.
    - Useful for flexible outputs when you expect the model to infer a reasonable structure based on your prompt.
-   - JSON mode is older and has been used by developers since early API implementations.
-
-- **Structured outputs (schema mode)** (deterministic/structured):
-   - Type `{"type": "json_schema"}`
-   - This mode enforces a strict schema format, where the output adheres to the predefined structure.
-   - Supports complex types and validation mechanisms as per the [JSON schema specification](https://json-schema.org/specification/).
-   - Structured outputs is a newer feature implemented by OpenAI in 2024 to enable stricter, schema-based response formatting.
+   - JSON mode is older and has been used by developers since early API implementations but lack reliability in response formats.
```

Comment:
```suggestion
   - JSON mode is older and has been used by developers since early API implementations, but lacks reliability in response formats.
```